java/org/openobservatory/measurement_kit/android/ResourceUtils.java

```diff
@@ -0,0 +1,68 @@
+// Part of measurement-kit <https://measurement-kit.github.io/>.
+// Measurement-kit is free software. See AUTHORS and LICENSE for more
+// information on the copying conditions.
+
+package org.openobservatory.measurement_kit.android;
+
+import android.app.Activity;
+import android.util.Log;
+
+import java.io.FileOutputStream;
+import java.io.InputStream;
+
+public class ResourceUtils {
+    private static final String LOG_TAG = "ResourceUtils";
+    private static final String CA_BUNDLE = "cacert.pem";
+    private static final String GEOIP = "geoip.dat";
+    private static final String GEOIP_ASNUM = "geoipasnum.dat";
+
+    public static void copy_ca_bundle(Activity activity, int id) {
+        copy_resource(activity, id, CA_BUNDLE);
+    }
+
+    public static void copy_geoip(Activity activity, int id) {
+        copy_resource(activity, id, GEOIP);
+    }
+
+    public static void copy_geoip_asnum(Activity activity, int id) {
+        copy_resource(activity, id, GEOIP_ASNUM);
+    }
+
+    public static void copy_resource(Activity activity, int id, String filename) {
```

Comment:
I'd open an issue for that. Future versions of measurement-kit do not probably need to ship with geoip and tests lists as resources, because we have developed code to download resources automatically and that should land in the v0.4.x branch. This leaves as the only mandatory resource to be copied the SSL certificates bundle. I think it is possible to read the resource as a text or byte buffer and then pass it to a function, right? In such case, we can add to MK and API by which we pass the certificates bundle as a string rather than as a path.